src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/Binders/EnumTypeModelBinder.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Globalization;
+
+namespace Microsoft.AspNetCore.Mvc.ModelBinding.Binders
+{
+    /// <summary>
+    /// <see cref="IModelBinder"/> implementation to bind models for types deriving from <see cref="Enum"/>.
+    /// </summary>
+    public class EnumTypeModelBinder : SimpleTypeModelBinder
+    {
+        private readonly bool _allowBindingUndefinedValueToEnumType;
+
+        public EnumTypeModelBinder(bool allowBindingUndefinedValueToEnumType, Type modelType)
+            : base(modelType)
+        {
+            if (modelType == null)
```

Comment:
Don't need to repeat checks the base constructor does, especially when the parameter isn't used here.